### PR TITLE
[ACI] Onboard USDC.e on Gnosis

### PR DESCRIPTION
**Pull request by Rozengarden**

### CODE REVIEW

File: src/20240717_AaveV3Gnosis_OnboardUSDCEOnGnosis/AaveV3Gnosis_OnboardUSDCEOnGnosis_20240717.sol [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    AaveV3Gnosis.POOL.supply(USDCe, USDCe_SEED_AMOUNT, address(AaveV3Gnosis.COLLECTOR), 0);
+  }
+
+  function newListings() public pure override returns (IAaveV3ConfigEngine.Listing[] memory) {
+    IAaveV3ConfigEngine.Listing[] memory listings = new IAaveV3ConfigEngine.Listing[](1);
+
+    listings[0] = IAaveV3ConfigEngine.Listing({
+      asset: USDCe,
+      assetSymbol: 'USDCe',
+      priceFeed: 0x26C31ac71010aF62E6B486D1132E266D6298857D,
```
Comment (brotherlymite) [inline src/20240717_AaveV3Gnosis_OnboardUSDCEOnGnosis/AaveV3Gnosis_OnboardUSDCEOnGnosis_20240717.sol:34]: @Rozengarden I think we should use the capo adapter in this case, we can re-use the existing one as we are using the same chainlink `USDC / USD` feed 
https://github.com/bgd-labs/aave-address-book/blob/main/src/AaveV3Gnosis.sol#L180

File: src/20240717_AaveV3Gnosis_OnboardUSDCEOnGnosis/OnboardUSDCEOnGnosis.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+This proposal aims to onboard USDC.e into the Aave V3 pools on Gnosis in support of this migration, allowing Aave users to transition to the new version of USDC.
+
+## Specification
+
+The table below illustrates the configured risk parameters for **USDCe**
+
+| Parameter                          |                                      Value |
+| ---------------------------------- | -----------------------------------------: |
+| Isolation Mode                     |                                       true |
```
Comment (brotherlymite) [inline src/20240717_AaveV3Gnosis_OnboardUSDCEOnGnosis/OnboardUSDCEOnGnosis.md:24]:
```suggestion
| Isolation Mode                     |                                       false |
```

File: src/20240717_AaveV3Gnosis_OnboardUSDCEOnGnosis/AaveV3Gnosis_OnboardUSDCEOnGnosis_20240717.sol (hunk 1 above)
Comment (marczeller) [inline src/20240717_AaveV3Gnosis_OnboardUSDCEOnGnosis/AaveV3Gnosis_OnboardUSDCEOnGnosis_20240717.sol:34]:
```suggestion
      priceFeed: 0x0a2d05bc646C65A029e602c257DfA14adF8BfAd2,
```

Comment (marczeller) [reply src/20240717_AaveV3Gnosis_OnboardUSDCEOnGnosis/AaveV3Gnosis_OnboardUSDCEOnGnosis_20240717.sol]: done

File: src/20240717_AaveV3Gnosis_OnboardUSDCEOnGnosis/OnboardUSDCEOnGnosis.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+| Stable Borrowing                   |                                   DISABLED |
+| Stable Slope1                      |                                        9 % |
+| Stable Slope2                      |                                       75 % |
+| Base Stable Rate Offset            |                                        0 % |
+| Stable Rate Excess Offset          |                                        0 % |
+| Optimal Stable To Total Debt Ratio |                                        0 % |
+| Flashloanable                      |                                    ENABLED |
+| Siloed Borrowing                   |                                   DISABLED |
+| Borrowable in Isolation            |                                   DISABLED |
+| Oracle                             | 0x26C31ac71010aF62E6B486D1132E266D6298857D |
```
Comment (brotherlymite) [inline src/20240717_AaveV3Gnosis_OnboardUSDCEOnGnosis/OnboardUSDCEOnGnosis.md:48]:
```suggestion
| Oracle                             | 0x0a2d05bc646C65A029e602c257DfA14adF8BfAd2 |
```

**Review by pavelvm5 — CHANGES_REQUESTED** [review]

**Comment by marczeller** [comment]
@pavelvm5 all comments have been addressed. thanks for feedback.

File: .assets/114dc68953c76e2f28a20e6823526478f668d066.svg
Comment (pavelvm5) [inline .assets/114dc68953c76e2f28a20e6823526478f668d066.svg:1]: Could you remove excess svg files from PR?
This is the first

File: .assets/543dace5ab86891e42b5a6d57a11e7f8f269b3e8.svg
Comment (pavelvm5) [inline .assets/543dace5ab86891e42b5a6d57a11e7f8f269b3e8.svg:1]: This too

File: .assets/5cccf3b69ba9b87d4427fd74090033d5fca78f1d.svg
Comment (pavelvm5) [inline .assets/5cccf3b69ba9b87d4427fd74090033d5fca78f1d.svg:1]: This too

File: .assets/5fed329e8856dde8915ff9f398caf93d5f851669.svg
Comment (pavelvm5) [inline .assets/5fed329e8856dde8915ff9f398caf93d5f851669.svg:1]: This too

File: .assets/9288405343690aed3409dfae4e389cc38a583b94.svg
Comment (pavelvm5) [inline .assets/9288405343690aed3409dfae4e389cc38a583b94.svg:1]: This too

File: .assets/c6b230503b2de3bc966e0ba8b5ff3dfd2f42fdb8.svg
Comment (pavelvm5) [inline .assets/c6b230503b2de3bc966e0ba8b5ff3dfd2f42fdb8.svg:1]: This too

File: .assets/d7b7613b07b73e6e47d0e203c2f438aeef747a01.svg
Comment (pavelvm5) [inline .assets/d7b7613b07b73e6e47d0e203c2f438aeef747a01.svg:1]: This too, the last one

**Review by pavelvm5 — CHANGES_REQUESTED** [review]
Some excess files are added in the last commit, could you delete them?

**Review by brotherlymite — APPROVED** [review]

**Review by pavelvm5 — APPROVED** [review]

**Pull request merged**